Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -202,7 +202,7 @@ for this property.
 - (WKNavigation *)_restoreSessionState:(_WKSessionState *)sessionState andNavigate:(BOOL)navigate;
 - (_WKSessionState *)_sessionStateWithFilter:(BOOL (^)(WKBackForwardListItem *item))filter;
 
-@property (nonatomic, setter=_setAllowsRemoteInspection:) BOOL _allowsRemoteInspection;
+@property (nonatomic, setter=_setAllowsRemoteInspection:) BOOL _allowsRemoteInspection WK_API_DEPRECATED_WITH_REPLACEMENT("inspectable", macos(WK_MAC_TBA, WK_MAC_TBA), ios(WK_MAC_TBA, WK_IOS_TBA));
```

Comment:
I think this should be `WK_API_DEPRECATED_WITH_REPLACEMENT("inspectable", macos(10.10, WK_MAC_TBA), ios(8.0, WK_IOS_TBA))`